Adafruit_DAP.cpp

```diff
@@ -45,6 +45,26 @@ bool Adafruit_DAP::begin(int swclk, int swdio, int nreset, ErrorHandler perr) {
   // return true;
 }
 
+void Adafruit_DAP::targetConnect() {
+  Serial.println("Connecting...");
+  // First disconnect, in case this really is e reconnect
+  if (!dap_disconnect())                      perror_exit(error_message);
+
+  char debuggername[100];
+  if (!dap_get_debugger_info(debuggername))   perror_exit(error_message);
+  Serial.print(debuggername); Serial.print("\n\r");
```

Comment:
the targetConnect method replaces a lot of boilerplate code, being exactly equal in all the sketches. Without these serial prints, the targetonnect will not have the same functionality as the boilerplate code, and the serial prints must be put back into the sketches.

Reply:
The duplication in the example sketches aren't much of the issue since they are trying doing the same thing just on different boards and platforms. In general, I wouldn't want library to print anything else except for the error/warning message. This is minor though, I could do a follow up to change it later on.

Reply:
No problem. I agree with your view.